Mathlib/CategoryTheory/DiscreteCategory.lean

```diff
@@ -9,27 +9,30 @@ Authors: Stephen Morgan, Scott Morrison, Floris van Doorn
 ! if you have ported upstream changes.
 -/
 import Mathlib.CategoryTheory.EqToHom
-import Mathlib.Data.Ulift
+import Mathlib.Data.ULift
+import Mathlib.Tactic.CasesM
+
+open Lean Meta
```

Comment:
This should go after the module docstring. Also, I would hide it into a `section` or `open .. in` if it is only needed in 1 or 2 declarations.

Reply:
I think this is from an older version of the file?